libs/wire-api/src/Wire/API/MLS/CipherSuite.hs

```diff
@@ -122,7 +123,13 @@ data CipherSuiteTag
   deriving (Arbitrary) via (GenericUniform CipherSuiteTag)
 
 defCipherSuite :: CipherSuiteTag
-defCipherSuite = MLS_128_DHKEMX25519_AES128GCM_SHA256_Ed25519
+defCipherSuite = MLS_128_DHKEMP256_AES128GCM_SHA256_P256
```

Comment:
This is unused now. As discussed on the call, we should delete this and make a v8 version of the only endpoint which uses `defaultCipherSuiteV7` which makes the ciphersuite mandatory.

Reply:
Also feel free to do this in a separate PR/ticket.

Reply:
Now done in this PR.